mbedtls/Cargo.toml

```diff
@@ -35,11 +35,15 @@ rs-libc = "0.2.0"
 chrono = "0.4"
 
 [dependencies.mbedtls-sys-auto]
-version = "2.25.0"
+version = "2.28.0"
```

Comment:
Why are you bumping the dependency?

Reply:
I think, this is just a fix. The version number under ./mbedtls-sys-auto is  `2.28.0`

Reply:
This line specifies the minimum version required. This code doesn't make any changes to MbedTLS features used, so it doesn't make sense that the requirement is changed?

Reply:
you're correct